config/pipeline.yaml

```diff
@@ -552,3 +653,21 @@ build_configs:
     tree: mediatek
     branch: 'for-next'
     variants: *build-variants
+
+  android_mainline:
+    tree: android
+    branch: 'android-mainline'
+    build_environment: gcc-10
+    arch: armel
+    defconfig: multi_v7_defconfig
+    filters:
+      - regex: { defconfig: 'multi_v7_defconfig' }
+
+  android_mainline_imx:
+    tree: android
+    branch: 'android-mainline'
+    build_environment: gcc-10
+    arch: armel
+    defconfig: imx_v6_v7_defconfig
+    filters:
+      - regex: { defconfig: 'imx_v6_v7_defconfig' }
```

Comment:
This can go away, as explained above this will be covered by the `android_mainline` config. A separate `kbuild` job should be created using this defconfig.